crates/networking/p2p/net.rs

```diff
@@ -760,6 +894,34 @@ async fn pong(socket: &UdpSocket, to_addr: SocketAddr, ping_hash: H256, signer:
     }
 }
 
+async fn send_enr_request(
+    socket: &UdpSocket,
+    to_addr: SocketAddr,
+    signer: &SigningKey,
+) -> Option<H256> {
+    let mut buf = Vec::new();
+
+    let expiration: u64 = (SystemTime::now() + Duration::from_secs(20))
+        .duration_since(UNIX_EPOCH)
+        .unwrap_or_default()
+        .as_secs();
+
+    let enr_req = discv4::Message::ENRRequest(ENRRequestMessage::new(expiration));
+
+    enr_req.encode_with_header(&mut buf, signer);
+    let bytes_sent = socket.send_to(&buf, to_addr).await;
+
+    let Ok(bytes_sent) = bytes_sent else {
+        return None;
+    };
```

Comment:
If "mapping" the `Err` to a `None` is ok, then I suggest a little refactor:
```suggestion
    let bytes_sent = socket.send_to(&buf, to_addr).await.ok()?;
```